### PR TITLE
Automation and updation of documentation

### PR DESCRIPTION
**Pull request by ravisaxena23**
**Fixes**
1. New commit of readme.md does not direct to demo of vue-vocabulary site.
2. readme.md contains recurssion it calls same repo when you click link in contribution
 by @ravisaxena23
3 updated instruction.md as automated linting. reducing the pain of developers finger(haha) also reducing the headache of mentor as someone might forget for quality checking.

**Description**
update readme.md and instruction.md

**Type of PR** 
This PR is a [feature|hotfix|refactor].

**Technicalities**

**Tests**
Check readme.md and instruction.md
run npm run build or npm run story book quality check is done automatically.

**Checklist:**
 
- [x] My pull request has a descriptive title (not a vague title like "Update `index.md`").
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x] I added tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

**Legal**
By submitting this PR, I agree to abide by the terms of the Developer 
Certificate of Origin.

<details>
<summary>Developer Certificate of Origin</summary>

Developer Certificate of Origin<br>
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.<br>
1 Letterman Drive<br>
Suite D4700<br>
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
</details>

### CODE REVIEW

**Comment by dhruvkb** [comment]
Replying to your points:
1. That is a different issue from the rest of the points and should be a different PR. We strive to keep PRs independent of each other as that allows them to be reviewed, updated and merged asynchronously, while also increasing your PR count so win-win!
2. The `README.md` file was also present as the Styleguidist homepage, that link should be removed. Your PR should only make this very surgical precise change. Good observation, nice find and fair point 👍. 
3. Linting is a separate task for a reason. Many a times we just want to run quick tests and linting such short-lived code is a waste of effort. Since ESLint runs on maximum strictness, all inconsistencies are reported as errors, which will prevent the next task from running. Linting should be run as a git commit hook instead of with every script. Also the possiblilty of forgetting to check linting is non-existent as CircleCI does test `npm run lint` on every commit and every PR.

File: INSTRUCTIONS.md [hunk 1]
```diff
@@ -75,10 +75,9 @@ $ git checkout -b branch_name
 Resolve the issue by changing the code. Update tests if need be.
 
 **Step 3:**
-Run the `lint` and `test:unit` tasks to ensure code quality and functionality.
+Run the `test:unit` tasks to ensure code quality and functionality.
 
 ```
-$ npm run lint
 $ npm run test:unit
 ```
```
Comment (dhruvkb) [inline INSTRUCTIONS.md]: This change will cause a lot of issues.

File: README.md [hunk 2]
```diff
@@ -63,7 +63,7 @@ Vocabulary is available as
 ### Demo
 
 The showcase of the true extent of Vocabulary can be found in the
-**[interactive storybook](https://cc-vue-vocabulary.netlify.com/storybook)**
+**[interactive storybook](https://cc-vue-vocabulary.netlify.com/)**
```
Comment (dhruvkb) [inline README.md]: See #180.

File: README.md [hunk 3]
```diff
@@ -82,10 +82,9 @@ contribute in some way.
   Documentation powered by Netlify.
 </p>
 
-### Contributing
+###  For Contribution
```
Comment (dhruvkb) [inline README.md]: No difference.

File: README.md [hunk 4]
```diff
@@ -82,10 +82,9 @@ contribute in some way.
...
 
-
-**[GitHub repository](https://github.com/creativecommons/vue-vocabulary)**
```
Comment (dhruvkb) [inline README.md]: 👍

File: README.md [hunk 5]
```diff
@@ -82,10 +82,9 @@ contribute in some way.
...
+**[Instruction Manual](INSTRUCTIONS.md)**
```
Comment (dhruvkb) [inline README.md]: Instead of this...

File: README.md [hunk 6]
```diff
@@ -103,8 +102,6 @@ project in a number of ways.
 | **Write**     | If you have a knack for writing technical articles, you could be the voice of the library's documentation, making it easy to use and understand. |
 | **Share**     | If you can't contribute in these ways, you can refer the project to a friend who might be able to. Spreading the word is the easiest way to help out. |
 
-If you're interested, read the file `INSTRUCTIONS.md` in the
```
Comment (dhruvkb) [inline README.md]: ...add the link here.

File: package.json [hunk 7]
```diff
@@ -7,8 +7,8 @@
     "test:unit": "vue-cli-service test:unit",
     "lint": "vue-cli-service lint",
     "build": "node library/build.js",
-    "storybook": "vue-cli-service storybook:serve -s ./src/assets -p 8080",
-    "build:documentation": "vue-cli-service storybook:build -s ./src/assets -o ./docs"
+    "storybook": "npm run lint && vue-cli-service storybook:serve -s ./src/assets -p 8080",
+    "build:documentation": " npm run lint && vue-cli-service storybook:build -s ./src/assets -o ./docs"
```
Comment (dhruvkb) [inline package.json]: This change would cause problems for developers.

File: README.md (hunk 3 above)
Comment (ravisaxena23) [reply README.md]: it shows manual you have to follow for contribution contributing does not give any meaning

File: README.md (hunk 4 above)
Comment (ravisaxena23) [reply README.md]: why we need a link of same repo when we are already on it?

File: README.md (hunk 2 above)
Comment (ravisaxena23) [reply README.md]: https://cc-vue-vocabulary.netlify.com/storybook this shows error says page not found however
https://cc-vue-vocabulary.netlify.com/ this takes to the demo of website.

File: INSTRUCTIONS.md (hunk 1 above)
Comment (ravisaxena23) [reply INSTRUCTIONS.md]: I'm removing it

File: README.md (hunk 4 above)
Comment (ravisaxena23) [reply README.md]: understood

**Pull request closed without merging**